pkg/kubelet/network/dns/dns.go

```diff
@@ -397,6 +421,9 @@ func (c *Configurer) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error) {
 			dnsConfig.Servers = []string{}
 			for _, ip := range clusterDNS {
 				dnsConfig.Servers = append(dnsConfig.Servers, ip.String())
+				nodeInfoMsg := fmt.Sprintf("For 1 TP X 1 RP verification - ClusterDNS IP : %q ", ip.String())
+				c.recorder.Eventf(c.nodeRef, v1.EventTypeWarning, "GetingClusterDNS", nodeInfoMsg)
+				c.recorder.Eventf(pod, v1.EventTypeWarning, "GettingClusterDNS", "pod: %q. %s", format.Pod(pod), nodeInfoMsg)
```

Comment:
in scale-out env, single event recorder probably has difficulty to send events of node and pod to same place.

Reply:
Yes, good to know.